locations/spiders/DskBank.py

```diff
@@ -0,0 +1,66 @@
+import re
+
+import scrapy
+
+from locations.categories import Categories, apply_category
+from locations.hours import DAYS_BG, OpeningHours, sanitise_day
+from locations.items import Feature
+
+
+class DSKBankBGSpider(scrapy.Spider):
+    name = "dsk_bank_bg"
+    item_attributes = {"brand": "Банка ДСК", "brand_wikidata": "Q5206146"}
+    allowed_domains = ["https://www.dskbank.bg/"]
+    start_urls = ["https://dskbank.bg/контакти/клонова-мрежа/GetOffices/"]
+
+    def parse(self, response):
+        for data in response.json():
+            item = Feature()
+            item["ref"] = data["Id"]
+            item["name"] = data["Name"]
+            item["lat"] = data["Latitude"]
+            item["lon"] = data["Longitude"]
+            item["email"] = data["Email"]
+
+            branch_type = data["BranchType"]
+            if "Branch" in branch_type:
+                apply_category(Categories.BANK, item)
+            else:
+                apply_category(Categories.ATM, item)
+
+            text = data["Text"]
+            clean_text = re.sub(r"<.*?>", "\n", text)
+            clean_text = re.sub("\n+", "\n", clean_text)
+            clean_text = clean_text.strip("\n").split("\n")
+
+            item["addr_full"] = clean_text[0]
+
+            phone = data["Phone"]
+            if phone:
+                phone = re.sub(r"\(|\*|\|\);", "", phone)
+            item["phone"] = phone
```

Comment:
Phone numbers get cleaned up automatically by Scrapy middleware, and are formatted according to the particular country the phone number belongs within.

I think all you need here is `item["phone"] = data["Phone"].split(";", 1)[0]` ?